nixos/modules/config/stevenblack.nix

```diff
@@ -35,10 +47,19 @@ in
       default = [ ];
       description = "Additional blocklist extensions.";
     };
+
+    whitelist = mkOption {
+      type = types.listOf types.str;
```

Comment:
Could we use some naive regex instead of `types.str`? I'm a bit wary of people accidentally typing in something wrong and it doesn't immediately get caught, since it'll just lead to silent breakages in a lot of hard-to-debug places. Your sed pattern creation also generally expects a "." in the string, so let's enforce that at _minimum_.

Something super simple like `types.strMatching ".+\\.[^\\.]+"` should work.